src/local-ssh/ipc/extensionServiceServer.ts

```diff
@@ -121,22 +121,19 @@ class ExtensionServiceImpl implements ExtensionServiceImplementation {
 
     // TODO remove from protocol, don't pass sensitive info back and forth, only once for auth, daemon should do telemetry directly
     async sendLocalSSHUserFlowStatus(request: SendLocalSSHUserFlowStatusRequest, _context: CallContext): Promise<{}> {
+        if (!request.flowStatus || request.flowStatus === '') {
+            return {};
+        }
         const flow: UserFlowTelemetry = {
-            flow: 'ssh',
-            kind: 'local-ssh',
-            connType: request.connType === SendLocalSSHUserFlowStatusRequest_ConnType.CONN_TYPE_SSH ? 'ssh' : 'tunnel',
+            flow: 'local_ssh',
```

Comment:
same let's rename to `ssh_tunnel` or `ssh_proxy`

Reply:
I don't know why we decided to rename 🤷‍♂️  local ssh seems to be more descriptive than other two, we have other ssh proxies like SSH gateway.